pkg/resources/marketplace/manifestHelper.go

```diff
@@ -100,7 +100,7 @@ func GetCRDsFromManifestAsStringList(dir string, regex string, packageYaml strin
 			currentVersion, err = GetCurrentCSVFromManifest(packageYaml)
 		}
 		if err != nil {
-			logrus.Fatal("Error")
+			log.Fatal("Error ", err)
```

Comment:
I'm confused with this err log, we are logging an error at line 93, wonder if I'm missing something but can't see why are we returning it again, and the "Error " isn't very descriptive to me.

Reply:
I've updated the msg to "Error getting current csv from manifest" 
The log is necessary because of the statement on L100

Reply:
Makes sense, thanks!